sbnobj/Common/POTAccounting/classes_def.xml

```diff
@@ -11,7 +11,9 @@
    <version ClassVersion="11" checksum="4062206176"/>
    <version ClassVersion="10" checksum="3057205612"/>
   </class>
-  <class name="sbn::EXTCountInfo" ClassVersion="11">
```

Comment:
@wesketchum is there a procedure for this? I start at 10 and increment from there as I have seen in other examples

Reply:
My main issue is to avoid proliferation of the version numbers during development. If data hasn't been written in 'older' versions that needs to be re-read, then we can remove those versions.

So, in this case: if no one has ever made data with 'version 10' that we need to read back, then I'd advocate that we make version 11 --> 10, and 'start' from there. If that's not the case, what you've done is exactly right.

Reply:
(or, to this point, if version 12 needs to be kept or not ... if not, can replace 12 with 13, and drop line 16).